test-frame-log-collector/src/main/java/io/skodjob/testframe/LogCollector.java

```diff
@@ -151,6 +171,22 @@ public void collectFromNamespaceToFolder(String namespaceName, String folderPath
         }
     }
 
+    /**
+     * Method that collects YAML of cluster wide resources
+     * {@link #rootFolderPath} with {@param folderPath}.
+     *
+     * @param folderPath        folder path for the log collection
+     */
+    public void collectClusterWideResourcesToFolder(String folderPath) {
+        String path = LogCollectorUtils.getFolderPath(rootFolderPath, folderPath);
+        createLogDirOnPath(path);
+        clusterWideResources.forEach(resource -> {
+            writeDataToFile(Paths.get(path, resource + ".yaml").toString(),
+                    kubeCmdClient.exec(false, false,
```

Comment:
Because we have `getResourceAsYaml` in the kubeCmdClient, should we have one method for the non-namespaced resources as well?